pages/index.tsx

```diff
@@ -2,8 +2,15 @@ import type { NextPage } from 'next'
 import Head from 'next/head'
 import Image from 'next/image'
 import Link from 'next/link'
+import { MetamaskDelegatedAccess } from '../src/lib/MetamaskDelegatedAccess'
 import styles from '../styles/Home.module.scss'
 
+const metamask = new MetamaskDelegatedAccess()
+const connectMetamask = async () => {
+  const {client, address} = await metamask.connect()
+  console.log('metamask connected', address, client)
```

Comment:
Could we remove these console.logs or put them behind a dev env flag?

Reply:
since am not sure how to interact with the frontend this is the breadcrumb left on how to run the code for Metamask Connect

Reply:
No probs, not a big deal to leave it in.